poc_iot_verifier/src/loader.rs

```diff
@@ -196,7 +196,7 @@ impl Loader {
                     &self.pool,
                     &id,
                     &event.data,
-                    &datetime_from_epoch(event.timestamp),
+                    &event.timestamp.to_timestamp()?,
```

Comment:
and here we're just going to the default timestamp units? shouldn't these all be to the same level of precision (presumably millis or nanos)?

Reply:
as above, branch is stale, fixed in latest